### PR TITLE
Added wordpress application

### PR DESCRIPTION
**Pull request by SimonGurney**
Tested and generates logs for a basic wordpress application, which is needed for the BSides NCL CTF which is going to use pwnSpoof!

### CODE REVIEW

**Comment by SimonGurney** [comment]
Sorry for the big PR, lots of new supporting functionality required

File: wordlists.py [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+    "receive",
+    "scroll",
+    "search",
+    "send",
+    "share",
+    "type",
+    "upload",
+    "view",
+    "visit",
+]
```
Comment (SimonGurney) [inline wordlists.py:235]: These are not very good, we either replace them now or we create a new issues for the backlog?

Comment (SimonGurney) [reply wordlists.py:235]: These are meant to be SEO friendly page names, more like:
aboutus
contactus
blog

**Comment by danieloateslee** [comment]
Reviewed and approved

File: wordlists.py (hunk 1 above)
Comment (danieloateslee) [reply wordlists.py:235]: Reviewed and I have created an issue for the comments above

**Review by danieloateslee — APPROVED** [review]

**Pull request merged**